doc/wcmp/WCMP-test-plan.md

```diff
@@ -0,0 +1,295 @@
+        nexthop via 10.10.28.1 dev Ethernet108 weight 25
+        nexthop via 10.10.29.1 dev Ethernet112 weight 25
+```
+4. Use redis-cli to disable WCMP in the database directly
+```
+root@router:/home/admin# redis-cli -n 4 HSET "BGP_DEVICE_GLOBAL|STATE" "wcmp_enabled" "false"
+(integer) 0
+root@router:/home/admin# redis-cli -n 4 HGETALL "BGP_DEVICE_GLOBAL|STATE"
+1) "tsa_enabled"
+2) "false"
+3) "wcmp_enabled"
+4) "false"
+```
+5. Verify BGP link bandwidth extended community is deleted in FRR
+```
+router# show running-config bgp
+Building configuration...
+
+Current configuration:
+!
+route-map TO_BGP_PEER_V4 permit 100
+exit
+!
+route-map TO_BGP_PEER_V6 permit 100
+exit
+!
+end
+```
+
+### Test case #3 - Verify WCMP behavior when no link failure
+1. Enable WCMP and verify WCMP status
+```
+admin@router:~$ sudo config bgp device-global wcmp enabled
+```
+2. Verify the route weight value on remote host is 25, which means WCMP take effective
+```
+admin@router:~$ ip route show 100.0.0.1
+100.0.0.1 proto bgp metric 20
+        nexthop via 10.10.26.1 dev Ethernet100 weight 25
+        nexthop via 10.10.27.1 dev Ethernet104 weight 25
+        nexthop via 10.10.28.1 dev Ethernet108 weight 25
+        nexthop via 10.10.29.1 dev Ethernet112 weight 25
+```
+3. Sends traffic to destination 100.0.0.1
+4. Verify that traffic throughput on each nexthop is the same
+
+
+### Test case #4 - Verify WCMP behavior after link failure
+1. Enable WCMP and verify WCMP status
+```
+admin@router:~$ sudo config bgp device-global wcmp enabled
+```
+2. Verify the route weight value on remote host is 25, which means WCMP take effective
+```
+admin@router:~$ ip route show 100.0.0.1
+100.0.0.1 proto bgp metric 20
+        nexthop via 10.10.26.1 dev Ethernet100 weight 25
+        nexthop via 10.10.27.1 dev Ethernet104 weight 25
+        nexthop via 10.10.28.1 dev Ethernet108 weight 25
+        nexthop via 10.10.29.1 dev Ethernet112 weight 25
+```
+3. Shutdown one link between DUT and SPINE1
+4. Verify the weight value of the route has updated after link failure
+```
+admin@router:~$ ip route show 100.0.0.1
+100.0.0.1 proto bgp metric 20
+        nexthop via 10.10.26.1 dev Ethernet100 weight 16
+        nexthop via 10.10.27.1 dev Ethernet104 weight 16
+        nexthop via 10.10.28.1 dev Ethernet108 weight 33
+        nexthop via 10.10.29.1 dev Ethernet112 weight 33
+```
+5. Sends traffic to destination 100.0.0.1
+6. Verify that traffic throughput on each nexthop is based on weight value of the interface
+7. Recover the failed link between DUT and SPINE1
+8. Verify the route weight value on remote host is 25
+
+### Test case #5 - Verify configuration persists after warm/cold/fast reboot
+1. Enable WCMP on DUT, verify the config take effective
+2. DUT warm/cold/fast reboot
+3. Verify WCMP configuration persist after reboot
+4. Traffic test, verify the WCMP function still take effect
+
+### Test case #6 - Verify configuration persists after warm/cold/fast reboot
```

Comment:
@weiguo-nvidia the same topic twice. A typo?

Reply:
Update the case name